helm/idol-eductionserver/values.schema.json

```diff
@@ -196,15 +207,45 @@
                 },
                 "replicas" :{
                     "type": "integer"
+                },
+                "labels": {
+                    "additionalProperties": {
+                      "type": [
+                        "string",
+                        "null"
+                      ]
+                    },
+                    "description": "Additional labels applied to all objects (https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/)",
+                    "type": [
+                      "object",
+                      "null"
+                    ]
+                },
+                "annotations": {
+                    "additionalProperties": {
+                      "type": [
+                        "string",
+                        "null"
+                      ]
+                    },
+                    "description": "Additional annotations applied to deployment/statefulset (https://kubernetes.io/docs/concepts/overview/working-with-objects/annotations/)",
+                    "type": [
+                      "object",
+                      "null"
+                    ]
+                },
+                "serviceAccountName":{
+                    "type": "string",
+                    "description": "Optional serviceAccountName for the pods (https://kubernetes.io/docs/tasks/configure-pod-container/configure-service-account)"
                 }
             },
-            "required": [
+            "required":[
                 "idolImage",
                 "ingress",
                 "name",
                 "aciPort",
                 "servicePort",
-                "grammarPackage"
```

Comment:
Why was `grammarPackage` made non-required?

Reply:
`grammarPackage` is defined and required within the `eductionserver` object [helm/idol-eductionserver/values.schema.json#L264-L287](https://github.com/opentext-idol/idol-containers-toolkit/blob/5c13d71c10c087fffdb9c603ee477dd345d4a986/helm/idol-eductionserver/values.schema.json#L264-L287) - the occurrence here was just in the wrong place.